src/dbnode/persist/fs/write.go

```diff
@@ -649,8 +668,14 @@ func writeCheckpointFile(
 	if err := digestBuf.WriteDigestToFile(fd, digestChecksum); err != nil {
 		// NB(prateek): intentionally skipping fd.Close() error, as failure
 		// to write takes precedence over failure to close the file
-		fd.Close()
+		_ = fd.Close()
```

Comment:
nit; Can we drop the comment and just log these errors?

Reply:
Will use `MultiError` here as well (and drop the comment).